src/vs/workbench/contrib/terminal/browser/terminalView.ts

```diff
@@ -257,7 +257,7 @@ export class TerminalViewPane extends ViewPane {
 						return;
 					}
 				};
-				return new ActionViewItem(action, panelOnlySplitAction, { icon: true, label: false, keybinding: this._getKeybindingLabel(action) });
+				return new ActionViewItem(action, panelOnlySplitAction, { ...options, icon: true, label: false, keybinding: this._getKeybindingLabel(action) });
 			}
 			case TerminalCommandId.SwitchTerminal: {
 				return this._instantiationService.createInstance(SwitchTerminalActionViewItem, action);
```

Comment:
Do we need to pass on the `options` also to `SwitchTerminalActionViewItem` and `SingleTerminalTabActionViewItem`?

Reply:
not supported as they are ?SelectActionViewItem`s